cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -330,8 +330,14 @@ void SparsePoseGraph::HandleWorkQueue() {
         for (const Constraint& constraint : result) {
```

Comment:
Instead of a comment: pull this out into a separate function `UpdateTrajectoryConnectivity(const sparse_pose_graph::ConstraintBuilder::Result& result)`?

Reply:
Done